liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java

```diff
@@ -105,8 +105,6 @@ public CommandResult run() throws Exception {
         commandScope.addArgumentValue(InternalDropAllCommandStep.HUB_CONNECTION_ID_ARG, this.hubConnectionId);
         commandScope.addArgumentValue(InternalDropAllCommandStep.SCHEMAS_ARG, this.schemas);
 
-        final CommandResults results = commandScope.execute();
```

Comment:
This line can't be removed, just the output variable

Reply:
done. reverted.